src/payment/strategies/moneris/moneris-payment-strategy.ts

```diff
@@ -39,7 +39,13 @@ export default class MonerisPaymentStrategy implements PaymentStrategy {
         }
 
         if (!this._iframe) {
-            this._iframe = this._createIframe(monerisOptions.containerId, initializationData.profileId, !!config.testMode);
+            const monerisLabels: MonerisHostedFieldsLabels = {
+                panLabel: initializationData.creditCardLabel,
+                expLabel: initializationData.expiryDateLabel,
+                cvdLabel: initializationData.cvdLabel,
+            };
+
+            this._iframe = this._createIframe(monerisOptions.containerId, initializationData.profileId, !!config.testMode, monerisLabels);
```

Comment:
This seems a bit of extra work, we are creating an object here and then creating the defaults inside the `_createIframe` function. Considering how much of the `initializationData` is needed in `_createIframe`, why don't we just pass it all in and do all the transformations inside it?